apps/client/src/features/viewers/studio/StudioClock.scss

```diff
@@ -1,182 +1,198 @@
+@use '../../../theme/ontimeColours' as *;
 @use '../../../theme/viewerDefs' as *;
 
 @font-face {
-  font-family: "digital-clock";
-  src: local('digital-7'), url('./../../../assets/fonts/digital-7.monoitalic.ttf') format('truetype') ;
+  font-family: 'seven-seg';
+  src:
+    local('DSEG7'),
+    url('./../../../assets/fonts/DSEG7ClassicMini-Regular.ttf') format('truetype');
 }
 
 /* =============== CLOCK STUFF ==================*/
 
-$clock-size: 90vh;
-$size-hours: min(3vh, 20px);
-$half-hours: min(1.5vh, 10px);
-$size-min: min(2.5vh, 18px);
-$half-min: min(1.25vh, 9px);
-$red-active: #c53030;
-$red-idle: #000000;
+$size-hours: clamp(12px, 2.5vw, 22px);
```

Comment:
don't know if we want to fix this but we only scale down the dots proportional to the width and not the height

![image](https://github.com/cpvalente/ontime/assets/19875125/b88d25d0-5ffd-4989-a661-57a5bba7591c)